build/dependencies.folderbuilds.props

```diff
@@ -0,0 +1,29 @@
+﻿<!-- This file is temporary while we rework dependencies management. It is used when building individual folders as if they were repos. -->
```

Comment:
How long will this need to exist? What about other folders (like DataProtection)?

Reply:
Nvrm I see data protection was also updated.

Reply:
Hopefully not long. I'm going to work over the next few days to cleanup dependency and reference management.